src/DIRAC/Core/Security/DiracX.py

```diff
@@ -82,7 +87,7 @@ class FutureClient:
 
 
 @contextmanager
-def DiracXClient() -> _DiracClient:
+def DiracXClient() -> Iterator[SyncDiracClient]:
```

Comment:
Why `Iterator`?

Reply:
As this function `yield`s an instance of `SyncDiracClient`. The previous type hint is wrong. The `contextmanager` decorator then transforms `Generator[T, None, None]` -> `ContextManager[T]`.

(`Iterator[T]` is an alias for `Generator[T, None, None]`)